apps/photos/src/components/PhotoViewer/FileInfo/MapBox.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import dynamic from 'next/dynamic';
+import { runningInBrowser } from 'utils/common';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.webpack.css'; // Re-uses images from ~leaflet package
+const L = runningInBrowser() ? require('leaflet') : null;
+runningInBrowser() ? require('leaflet-defaulticon-compatibility') : null;
+
+interface MapBoxProps {
+    location: { latitude: number; longitude: number };
+}
+
+const MapBox: React.FC<MapBoxProps> = ({ location }) => {
+    const [isClient, setIsClient] = useState(false);
+    const position: [number, number] = [location.latitude, location.longitude];
+
+    useEffect(() => {
+        setIsClient(true);
+    }, []);
+
+    useEffect(() => {
+        if (isClient) {
+            const mapContainer = document.getElementById('map-container');
+            if (mapContainer && !mapContainer.hasChildNodes()) {
+                const map = L.map(mapContainer).setView(position, 13);
```

Comment:
Should we increase the zoom level to 16/17/18? Can we please attach a screenshot in the PR summary for this particular change

Reply:
I have attached the screenshots for zoom level 13 and 16. It feels like zoom level 13 provides a more comprehensive view of the location